.circleci/config.yml

```diff
@@ -140,7 +140,6 @@ jobs:
           paths:
             - _build/default/rebar3_20.2.2_plt
       - run: make dialyzer
-      - run: make swagger-check
```

Comment:
@dincho This would need command `java` (Debian package `default-jre-headless`).
Currently Circle CI reuses the published Docker image for building the epoch package (`builder`). Options I see are:
A. Enrich `builder` image with java installation. This has drawback of unnecessarily polluting that Docker image used for building the epoch package;
B. Move CI to Docker image distinct from the one used for building epoch package;
C. Attempt to dynamically install java in Circle CI script.